native_client/ctcdecode/setup.py

```diff
@@ -40,33 +40,42 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+def maybe_rebuild(srcs, out_name, build_dir):
+    if not os.path.exists(out_name):
+        if not os.path.exists(build_dir):
+            os.makedirs(build_dir)
+
+        build_common(srcs=srcs,
+                     out_name=out_name,
+                     build_dir=build_dir,
+                     num_parallel=known_args.num_processes,
+                     debug=debug)
+
 project_version = read('../../VERSION').strip()
 
+CTC_DECODER_FILES = [
```

Comment:
Move this to `build_common.py` along with the other constants?

Reply:
I was unsure, so I'mfine

Reply:
Looks like you forgot to remove this.